xapi-profiles-communication.md

```diff
@@ -23,39 +23,75 @@ Part Two:	[xAPI Profiles Document Structure Specification](./xapi-profiles-struc
-      * 2.2. [Pattern Validation](./xapi-profiles-communication#2.2)
-  * 3.0. [Libraries](./xapi-profiles-communication#3.0)
+ * 1.0. [Profile Server](./xapi-profiles-communication#1.0)
+    * 1.1. [Profile Versions](./xapi-profiles-communication#profile_versions)
+    * 1.2. [Best Practices](xapi-profiles-communication#profile_server_best_practices)
+    * 1.3. [Example SPARQL Queries](xapi-profiles-communication#1.3)
+ * 2.0. [Algorithms](./xapi-profiles-communication#2.0)
+    * 2.1. [Statement Template Validation](./xapi-profiles-communication#2.1)
+    * 2.2. [Pattern Validation](./xapi-profiles-communication#2.2)
+ * 3.0. [Libraries](./xapi-profiles-communication#3.0)
+
 
 
 <a name="partthree"></a>
 # Part Three: Communication and Processing
 
-In addition to the ability to host profiles as documents, there will be infrastructure for querying and manipulating profiles. This document describes the algorithms for processing profiles, including the exact rules by which Statement Templates and Patterns are validated against Statements. It also describes a “Profile Server” to make it easier to manage and answer questions about profiles from a centralized location, including implementing the algorithms.
+In addition to the ability to host profiles as documents, there will be infrastructure 
+for querying and manipulating profiles. This document describes the algorithms for 
+processing profiles, including the exact rules by which Statement Templates and Patterns 
+are validated against Statements. It also describes a “Profile Server” to make it easier 
+to manage and answer questions about profiles from a centralized location, including 
+implementing the algorithms.
 
 ## <a name="1.0">1.0</a> Profile Server
 
-A Profile Server will
-* allow administrators to add profiles by their contents or by URI to the Profile Server
-* provide a review process people desiring to add profiles can submit to.
-* provide mechanisms to encourage contribution of Profile translations
-* encourage open licensing of Profiles
+A Profile Server manages xAPI Profiles from a centralized location. [An RDF triple store](https://en.wikipedia.org/wiki/Triplestore) is responsible for the storage of profiles. 
+A Profile Server will allow administrators to add profiles by their contents or by URI to the Profile Server
+
+A Profile Server will host a SPARQL endpoint containing the RDF information from the 
+contained profiles at the path /SPARQL. This enables xAPI Profiles to be queried. SPARQL 
+servers have the ability to divide information into multiple datasets, or graphs, and 
+offer separate querying on them. One of these is the default graph, which is queried 
+when no other graph is specified. The default graph at this SPARQL endpoint will 
+contain all the current versions of profiles. 
+
+### <a name="profile_versions">1.1</a> Profile Versions
+
+Every profile version will be in a [Named Graph](https://www.w3.org/TR/sparql11-query/#namedGraphs) 
```

Comment:
Hmm, I missed Profile capitalization here in my earlier pass, I should tidy that up too.